package.json

```diff
@@ -33,7 +33,7 @@
     "angular-bootstrap-npm": "jusopi/angular-bootstrap-npm#0.13.1",
     "angular-contenteditable": "0.0.2",
     "angular-dynamic-locale": "^0.1.32",
-    "angular-embed": "superdesk/angular-embed#2afd7397a7179f25b66c4b5f13ea2a0daf799821",
+    "angular-embed": "petrjasek/angular-embed#79c1038",
```

Comment:
great! BTW: I checked on http://sdcpr-1508.test.superdesk.org/ instance and it's working now :)